db/data/20200212151551_foo_bar_bag.rb

```diff
@@ -0,0 +1,3 @@
+# Run me with `rails runner db/data/20200212151551_foo_bar_bag.rb`
```

Comment:
Should this have been committed?

Reply:
Nope, there's a [fixup commit](https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/865/commits/5cd20dad33ad693ab1b9f7a21d5cf1759d96d122) that just needs squashing down.